uber/site_sections/preregistration.py

```diff
@@ -315,10 +318,13 @@ def process_group_payment(self, session, payment_id, stripeToken):
                 attendee.amount_paid = attendee.total_cost
                 session.merge(attendee)
 
-            if group.tables:
-                send_email(c.MARKETPLACE_EMAIL, c.MARKETPLACE_EMAIL, 'Dealer Payment Completed',
-                           render('emails/dealers/payment_notification.txt', {'group': group}), model=group)
             session.merge(group)
+            if group.tables:
+                try:
+                    send_email(c.MARKETPLACE_EMAIL, c.MARKETPLACE_EMAIL, 'Dealer Payment Completed',
+                               render('emails/dealers/payment_notification.txt', {'group': group}), model=group)
+                except:
+                    log.error('unable to send dealer payment confirmation email', exc_info=True)
             raise HTTPRedirect('group_members?id={}&message={}', group.id, 'Your payment has been accepted!')
```

Comment:
note: moved session.merge() to happen BEFORE sending the email.  this seems OK to me? only thing I can think of are presave_adjustments() would run first which........ seems correct?